Formula/soccerwindow2.rb

```diff
@@ -16,12 +16,17 @@ class Soccerwindow2 < Formula
   depends_on "librcsc"
 
   def install
+    # ld: framework not found Security-lz
+    inreplace "configure",
```

Comment:
Wow this bug does still exist! It's so persistent because the problem is in the m4 file.
https://sourceforge.net/p/sserver/code/HEAD/tree/rcss/trunk/rcssmonitor_qt4/m4/qt.m4